worlds/dkc3/Options.py

```diff
@@ -75,13 +75,55 @@ class PercentageOfBananaBirds(Range):
     default = 100
 
 
+class KONGsanity(Toggle):
+    """
+    Whether collecting all four KONG letters in each level grants a check
+    """
+    display_name = "KONGsanity"
+
+
 class LevelShuffle(Toggle):
     """
     Whether levels are shuffled
     """
     display_name = "Level Shuffle"
 
 
+class Difficulty(Choice):
+    """
+    Which Difficulty Level to use
+    NORML: The Normal Difficulty
+    HARDR: Many DK Barrels are removed
+    TUFST: Most DK Barrels and all Midway Barrels are removed
+    """
+    display_name = "Difficulty"
+    option_norml = 0
+    option_hardr = 1
+    option_tufst = 2
+    default = 0
+
+    @classmethod
+    def get_option_name(cls, value) -> str:
+        if cls.auto_display_name:
+            return cls.name_lookup[value].upper()
+        else:
+            return cls.name_lookup[value]
+
+
+class Autosave(DefaultOnToggle):
```

Comment:
Is there ever a reason to not have autosave? as in, why is this an option?

Reply:
There has been a single case of a softlock with this randomizer, which would have been permanent if autosave had been on. That is fixed and there hopefully isn't another, but that is one concern.
Also, the game calls you a "Cheatin' Chump" at the end if you use the autosave, which some players may not want.